src/connect-zbt-1/getting-started-with-zigbee/zbt-1-migrate-deconz-network.md

```diff
@@ -5,9 +5,10 @@ zendesk:
   position: 4
   labels: connect zbt-1, getting started, migrate, migration
 productName: "Home Assistant Connect&nbsp;ZBT-1"
+type: "zbt1"
 ---
 
-Follow this guide if you have a deCONZ network running with a ConBee II or RaspBee II and want to migrate to Connect&nbsp;ZBT-1 and {% abbr "ZHA" %}.
+Follow this guide if you have a deCONZ network running with a ConBee II or RaspBee II and want to migrate to {{ productName }} and the **Zigbee Home Automation** integration.
```

Comment:
This is missing the "Conbee III" reference. It's in the ZBT-2 article and also mentioned in this (ZBT-1) article later, but it's missing in the introduction here.